metabrainz/templates/users/tier.html

```diff
@@ -1,28 +1,28 @@
 {% extends 'base.html' %}
 
-{% block title %}{{ tier.name }} tier - MetaBrainz Foundation{% endblock %}
+{% block title %}{{ tier.name + _('tier') }} - MetaBrainz Foundation{% endblock %}
```

Comment:
This should be done the opposite way. Translators will only get `tier`, not `Some tier`.